package.json

```diff
@@ -50,6 +50,7 @@
     "del": "^3.0.0",
     "fs-extra": "^4.0.0",
     "glob": "^7.1.2",
+    "gulp": "^3.9.1",
```

Comment:
A gulptraum plugin shall never define `gulp` as its dependency.
It should be a peer dependency or none at all.

If there is a specific reason for the `gulp` dependency, we need to isolate it as much as possible.

Gulptraum is able to run on gulp@3, as well as gulp@4 and I want to keep it that way.